pkg/engine/engine.go

```diff
@@ -203,15 +203,15 @@ func (e Engine) initFunMap(t *template.Template) {
 		if val == nil {
 			if e.LintMode {
 				// Don't fail on missing required values when linting
-				log.Printf("[INFO] Missing required value: %s", warn)
+				slog.Warn("missing required value", "value", warn)
```

Comment:
In this case the `warn` is a message that's shared with the template executor when the required thing is not present. So, the key name could be something like `message`.